src/helpers/figures/charts/chart_common.ts

```diff
@@ -532,10 +532,19 @@ export function interpolateData(
     return [];
   }
   const labelMin = Math.min(...labels);
-  const labelMax = Math.max(...labels);
-  const labelRange = labelMax - labelMin;
-  const normalizedLabels = labels.map((v) => (v - labelMin) / labelRange);
-  const normalizedNewLabels = newLabels.map((v) => (v - labelMin) / labelRange);
+  // We change the normalization technique for logarithmic trend lines for a better fit
```

Comment:
Maybe you can explain a bit why it's better for the logarithmic model ? And also update your commit message as you're not filtering the "zeros" values anymore